sw/host/hsmtool/acorn/vendor/acorn.h

```diff
@@ -0,0 +1,1125 @@
+// Copyright lowRISC contributors (OpenTitan project).
```

Comment:
Are you able to remove this copyright header? Have you made any other modifications to the file at all?